src/js/tabs/cli.js

```diff
@@ -41,7 +41,35 @@ function getCliCommand(command, cliBuffer) {
     return commandWithBackSpaces(command, buffer, noOfCharsToDelete);
 }
 
-TABS.cli.initialize = function (callback) {
+function copyToClipboard(text, nwGui) {
+    function onCopySuccessful() {
+        writeLineToOutput("* " + i18n.getMessage("cliCopySuccessful"));
```

Comment:
Not sure if adding this to the output buffer is the best way of indicating that the copy worked - this means the output around it looks weird in subsequent copies, and the command sent after it is not commented any more.

How about changing the button text to 'Copied' for a couple of seconds or so instead?

Reply:
Done. Some CSS juggling was necessary to keep the width of the button when the text changes, because it looks better that way.